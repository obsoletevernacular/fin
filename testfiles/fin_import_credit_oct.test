Transaction ID,Posting Date,Effective Date,Transaction Type,Amount,Check Number,Reference Number,Description,Transaction Category,Type,Balance
"20171031 333333 333 333,333,333,333",43039,43036,Debit,-4.99,,30,Ext Credit Card Debit PLACE  LOCATION,AUTOCAT,Credit Card,0
"20171030 333333 333 333,333,333,334",43038,43034,Debit,-26,,31,Ext Credit Card Debit PLACE  LOCATION,,Credit Card,0
"20171030 333333 333 333,333,333,335",43038,43034,Debit,-5.5,,32,Ext Credit Card Debit PLACE  LOCATION,,Credit Card,0
"20171030 333333 333 333,333,333,336",43038,43034,Debit,-56.53,,33,Ext Credit Card Debit PLACE  LOCATION,,Credit Card,0
"20171028 333333 333 333,333,333,337",43036,43034,Debit,-11.37,,34,Ext Credit Card Debit PLACE  LOCATION,,Credit Card,0
"20171028 333333 333 333,333,333,338",43036,43034,Debit,-3.65,,35,Ext Credit Card Debit PLACE  LOCATION,,Credit Card,0
"20171026 333333 333 333,333,333,339",43034,43032,Debit,-17.29,,36,Ext Credit Card Debit PLACE  LOCATION,,Credit Card,0
"20171026 333333 333 333,333,333,340",43034,43032,Debit,-2.97,,37,Ext Credit Card Debit PLACE  LOCATION,,Credit Card,0
"20171026 333333 333 333,333,333,341",43034,43033,Debit,-34.86,,38,Ext Credit Card Debit PLACE  LOCATION,,Credit Card,0