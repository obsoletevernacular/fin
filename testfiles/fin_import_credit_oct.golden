1 files processed and stored to .default.obj
fin_import_credit_oct
    transactions: 9
    transactions_in: 0
    transactions_out: 9
    sum: -163.16
    sum_in: 0.00
    sum_out: -163.16
    avg: -18.13
1 files processed. summary report below:
summary
    transactions: 9
    transactions_in: 0
    transactions_out: 9
    sum: -163.16
    sum_in: 0.00
    sum_out: -163.16
    avg: -18.13
